### ISSUE TITLE
Tried to symmetrically pool BNB/RUNE but just ended up losing BNB from wallet ?

### ISSUE THREAD

**Issue by neb967** [asgardex/asgard-exchange#273]
Transferred from BNB from Binance and then swapped half for RUNE. 
Then tried to LP BNB/RUNE. But only the BNB was taken from my wallet ? 

My Thorchain addr: thor1z8r0c3azx3gxrl25kt6jhxd04jty2sq90m8206
My BEP2 addr: bnb12vjy92t2ldg3703qynxkeudxlg792dc855spzu

The RUNE is still in my Wallet, but the BNB has been transfered away to: bnb1ksxqxurvejkndenuv0alqawpr3e4vtqk855aal
(tx hash: E77C92F076F801D938B01F9893BFAA1293B7D614DBF8B3E58FC6F456FE895DD9)

**Comment by stuartkuentzel** [comment]
hey @neb967 :wave: it seems like a handful of people are experiencing the same issue. We're working on issues to resolve this, you can take a look at https://github.com/asgardex/asgard-exchange/issues/271 and https://github.com/asgardex/asgard-exchange/issues/259. Funds are safe, THORChain is just waiting on the RUNE side to be deposited, and since it fails, it's not listing you as a pool member.

**Comment by neb967** [comment]
> hey @neb967 👋 it seems like a handful of people are experiencing the same issue. We're working on issues to resolve this, you can take a look at #271 and #259. Funds are safe, THORChain is just waiting on the RUNE side to be deposited, and since it fails, it's not listing you as a pool member.

I see - is there a solution to this then ? I see you mentioning to resubmit the RUNE to pool it ?  Or should I resubmit the RUNE alone or try again to symmetrically pool RUNE/BNB but with a small amount to the same pool ? How do I know its the same BNB pool ?

**Comment by stuartkuentzel** [comment]
@neb967, we pushed one solution so it doesn't happen again.
at the moment, we're working on a solution to that has a UI to submit for the missing RUNE transaction. Will send you the link when it's pushed.

**Comment by neb967** [comment]
> @neb967, we pushed one solution so it doesn't happen again.
> at the moment, we're working on a solution to that has a UI to submit for the missing RUNE transaction. Will send you the link when it's pushed.

@stuartkuentzel  ok - will await the link over the coming days thanks

**Comment by nawtna** [comment]
I had a similar issue by trying to add liquidity on the ETH-RUNE pool—the ETH transaction went through, but not the RUNE one. Looking forward to try out the fix 👍

**Comment by stuartkuentzel** [comment]
A quick update regarding this issue. We have a pull request ready, we're waiting on a small update in the Midgard API on the THORChain end, which I guess will be next week sometime. Once that is updated, we will push our code, and you can recover your stuck pending funds.

**Comment by stuartwk** [comment]
Hey @nawtna, @neb967, would you mind trying out https://app.asgard.exchange/deposit-sym-recovery? Search for the corresponding pool where you have stuck assets.

What this does is, uses a RUNE Withdraw transaction to withdraw all funds from that pool (including successfully pooled funds). This should free up the pending funds. Something to keep in mind, due to some updates, wallets must have at least 3 RUNE to run RUNE transactions. It doesn't use all of this as gas, but creates a safety net for the transaction to go through.

If the withdraw transaction fails, please don't try it over again, and instead just leave a comment here.

**Comment by nawtna** [comment]
@stuartwk The withdraw worked ([transaction](https://viewblock.io/thorchain/tx/72770D8B1598C737B68CD58E60B6877D696BE3D952D8CBD3C1DF19690346F65E)), but it didn't withdraw all ETH that I added for liquidity. I initially added 0.099ETH to the pool ([transaction](https://etherscan.io/tx/0xaacc52c937d45acd245a7dbb5e145a32a3b88908cee791100e3c9b0d7ff47892)), but got 0.0528ETH back. When I check my [ETH address](https://etherscan.io/address/0xfed2035b4ab9bbf261428ba709c0848437311766) I don't see the withdraw transaction.

Let me know if there is something more that I can provide.

**Comment by stuartwk** [comment]
@nawtna https://etherscan.io/address/0xfed2035b4ab9bbf261428ba709c0848437311766#internaltx it looks like there's an "internal transaction" from about 20 minutes ago for 0.0462 Ether, which I believe would add up to about the 0.099 (minus network fees)?

**Comment by nawtna** [comment]
@stuartwk indeed 👍  Sorry, what is the internal transaction? Was it triggered by the Thor network itself?

**Comment by stuartwk** [comment]
@nawtna I'm not an ETH expert, but I'm sure it has to do with the ethereum smart contract THORChain has for processing ETH/ERC20 transactions. If you're interested, you can take a look at https://gitlab.com/thorchain/ethereum/eth-router/-/tree/master/contracts

**Comment by nawtna** [comment]
@stuartwk that wouldn't be the first time that I pay crazy fees with ETH 😅

Thanks for the update 👍

**Comment by stuartwk** [comment]
Closing this issue out as it seems to be working for users. @neb967 if you have issues with this, please feel free to open a new issue.
